tests/ssh-lua-rules/test.yaml

```diff
@@ -0,0 +1,15 @@
+pcap: ../ssh-banner-only/input.pcap
+
+requires:
+  min-version: 8
+
+args:
+  - --set security.lua.allow-rules=true -k none
```

Comment:
`--set security.lua.allow-rules=true` shouldn't be needed

Reply:
right